Doc/library/functions.rst

```diff
@@ -657,11 +657,14 @@ are always available.  They are listed here in alphabetical order.
       sign: "+" | "-"
       infinity: "Infinity" | "inf"
       nan: "nan"
-      numeric_value: `floatnumber` | `infinity` | `nan`
+      number: floatnumber [`exponent`]
```

Comment:
Nitpick: can we shuffle things around so that the dependence is strictly from top to bottom? (E.g., the definition of `digitpart` preceding that of `floatnumber` and the definition of `floatnumber` preceding that of `number`.)

It's a little tempting to do some renaming while we're here, but it's probably worth keeping consistency with the grammar productions.